package/lib/src/controls/app_bar.dart

```diff
@@ -62,3 +67,81 @@ class AppBarControl extends StatelessWidget implements PreferredSizeWidget {
   @override
   Size get preferredSize => Size.fromHeight(height);
 }
+
+class BottomAppBarControl extends StatefulWidget {
```

Comment:
Move `BottomAppBarControl` class to a separate `bottom_app_bar.dart` file please.